infrastructure/templates/template.json

```diff
@@ -1751,12 +1728,15 @@
         "PublicAppUrl": "[variables('publicAppUrl')]",
         "IdentityServer:IssuerUri": "[concat('urn=', parameters('adminUri'))]",
         "IdentityServer:Key:Name": "[concat('CN=', parameters('adminUri'))]",
-        "OpenIdConnect:Authority": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-openidconnect-authority'), '2018-02-14').secretUriWithVersion, ')')]",
-        "OpenIdConnect:ClientId": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-openidconnect-clientid'), '2018-02-14').secretUriWithVersion, ')')]",
-        "OpenIdConnect:ClientSecret": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-openidconnect-clientsecret'), '2018-02-14').secretUriWithVersion, ')')]",
-        "OpenIdConnectSpaClient:AllowOfflineAccess": "[parameters('openIdConnectSpaClientAllowOfflineAccess')]",
-        "OpenIdConnectSpaClient:RefreshTokenUsage": "[parameters('openIdConnectSpaClientRefreshTokenUsage')]",
-        "OpenIdConnectSpaClient:RefreshTokenExpiration": "[parameters('openIdConnectSpaClientRefreshTokenExpiration')]",
+        "OpenIdConnectIdentityFramework:ClientId": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-openidconnect-clientid'), '2018-02-14').secretUriWithVersion, ')')]",
+        "OpenIdConnectIdentityFramework:Authority": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-openidconnect-authority'), '2018-02-14').secretUriWithVersion, ')')]",
+        "OpenIdConnectIdentityFramework:TokenValidationParameters:ValidAudience": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-openidconnect-valid-audience'), '2018-02-14').secretUriWithVersion, ')')]",
+        "OpenIdConnectIdentityFramework:TokenValidationParameters:ValidIssuers": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-openidconnect-valid-issuers'), '2018-02-14').secretUriWithVersion, ')')]",
+        "OpenIdConnectAdminApiExtras:AdminApiScopeForAudience": "[concat('@Microsoft.KeyVault(SecretUri=', reference(variables('ees-openidconnect-audience-scope-name'), '2018-02-14').secretUriWithVersion, ')')]",
```

Comment:
Hey there,

Thanks, I've created ticket https://dfedigital.atlassian.net/browse/EES-4894 for this.

Another thing is that I've removed the configuration for the scope name that the backend looks for, because a) it's ALWAYS "access-admin-api" and b) various tests have to explicitly be fed this string in their InlineData setups anyway, so it wouldn't make sense for it to change given that it's a fundamental piece of the codebase.  Where possible though, I've referenced it as SecurityScopes.AccessAdminApiScopes constant.  There's therefore now no longer a need for the `ees-openidconnect-audience-scope-name` parameter, so I've removed it ta.  We'll still need the fully-qualified name one that the SPA must use when requesting tokens from Entra ID though, as that'll change per environment.

As a followup ticket after everything is seen to be working OK, I've also created ticket https://dfedigital.atlassian.net/browse/EES-4895 for removing old unused parameters.

In addition, I've also raised other tickets for various pieces of manual work in getting this out of the door:

* Updating and creating App Registrations - https://dfedigital.atlassian.net/browse/EES-4892
* Issuing instructions to dev / test test post-merge - https://dfedigital.atlassian.net/browse/EES-4896
* Moving final 2 files in "Areas" folder after EES-4814 is merged - https://dfedigital.atlassian.net/browse/EES-4897

Ta!